### DISCUSSION TITLE
YimMenu: [Object Spawn]

### DISCUSSION THREAD

**Discussion by mvcula** [YimMenu/YimMenu#2720] [Q&A] [answered]
I have a question, its possible spawning some objects like a tree, box, lamp etc? what script i must use be that working

**Comment by 99Anvar99** [comment, marked as answer]
> I have a question, its possible spawning some objects like a tree, box, lamp etc? what script i must use be that working

Yes, you will need the hash of that object,
lets say "Tree" for example: 2044426993
here is the website where you can find all objects: https://forge.plebmasters.de/objects

not sure if Yim Menu has the direct object spawning

**Reply by mvcula** [reply]
thanks =) will try
